pyne/mesh.py

```diff
@@ -1232,21 +1227,13 @@ def tag_cell_fracs(self, cell_fracs):
         for ve in range(num_vol_elements):
             max_cell_number = max(max_cell_number,
                                   len(cell_fracs[cell_fracs['idx'] == ve]))
-        # set the cell_number_tag & cell_fracs_tag
-        self.cell_number_tag = IMeshTag(max_cell_number, int, mesh=self,
-                                        name='cell_number_tag')
-        self.cell_fracs_tag = IMeshTag(max_cell_number, float, mesh=self,
-                                       name='cell_fracs_tag')
-        self.cell_largest_frac_number_tag = \
-            IMeshTag(1, int, mesh=self, name='cell_largest_frac_number')
-        self.cell_largest_frac_tag = IMeshTag(1, float, mesh=self,
-                                              name='cell_largest_frac')
+
+        # creat tag frame with default value
         cell_largest_frac_number = [-1] * num_vol_elements
         cell_largest_frac = [0.0] * num_vol_elements
-
-        # fill the data
         voxel_cell_number = np.empty(shape=(num_vol_elements,max_cell_number))
         voxel_cell_fracs = np.empty(shape=(num_vol_elements,max_cell_number))
+        # set the data
         for ve in range(num_vol_elements):
```

Comment:
The `ve` here doesn't represent an entity volume element, it more like the `idx` of the volume element. This `ve` here are [0, 1, 2, ...]. @pshriwise Do you think it's better to replace this `ve` with `i` or `idx`?

Reply:
Ah, my misunderstanding. Sorry. I think changing the variable might help with clarity here, yes. 

Also I'd recommend taking a look at the NumPy `fill` command. It could replace the first couple lines of this loop I think.

Reply:
I replaced the `ve` with `i` in these loop. And the Numpy `fill` command was applied to replace two lines of that loop.

Reply:
Nice! Looks good.